### ISSUE TITLE
all possible pos tags

### ISSUE THREAD

**Issue by pskadasi** [murali1996/CodemixedNLP#1]
Its not a issue but I want to ask what all possible pos tags are available and I need their abbreviations too. please let me know.
In the paper and this github repo, it isn't available.

**Comment by murali1996** [comment]
Hey @pskadasi , (with reference to [page 2 last paragraph of our paper](https://arxiv.org/pdf/2106.06004.pdf#page=2)) you can use any set of tags during fine-tuning. There is no restriction on the size of the set nor there is any predefined set of tags to use. Your custom tags must be provided along with the labels of your original task during the fine-tuning (for example, if your input sentence is "Word1 Word2 Word3 ... WordN", then your tags need to be space-separated something like "Tag1 Tag3 Tag1 .... TagK"). The set of the tags is obtained from all of the tags found in training data and embedding lookup is created accordingly.
